assets/jsons/generosity_challenge/chat_scripts/chat_script_day_0.json

```diff
@@ -24,21 +24,21 @@
             "answerText": "Okay",
             "next": {
               "type": "inputAnswer",
-              "inputAnswerType" : "text",
+              "inputAnswerType": "text",
               "side": "user",
               "text": "First name of parent",
               "saveKey": "firstName",
               "hidden": true,
               "next": {
                 "type": "inputAnswer",
-                "inputAnswerType" : "email",
+                "inputAnswerType": "phone",
                 "side": "user",
-                "text": "E-mail address of parent",
-                "saveKey": "email",
+                "text": "Phone number",
```

Comment:
Are we getting the email somewhere else now? Because we still need it for the registration api call but i can't seem to find where we're asking for it now in this PR.

Reply:
yes, in the redirect screen see _goToChallenge